lib/tests/shareable.test.js

```diff
@@ -0,0 +1,132 @@
+const mongoose = require('mongoose');
+const { MongoMemoryServer } = require('mongodb-memory-server');
+const request = require('supertest');
+const app = require('../app');
+const Shareable = require('../models/Shareable');
+
+describe('app routes', () => {
+    const mongo = new MongoMemoryServer();
+    beforeAll(() => {
+        return mongo.getUri()
+            .then(uri => mongoose.connect(uri, {
+                useUnifiedTopology: true,
+                useNewUrlParser: true
+            }));
+    });
+
+    beforeEach(() => {
+        return mongoose.connection.dropDatabase();
+    });
+
+    afterAll(() => {
+        return mongoose.connection.close();
+    });
```

Comment:
Your tests aren't finishing on GitHub because your MongoMemoryServer isn't stopping. Add this line and your tests GitHub will be happy...

return mongoose.connection.close()
      .then(mongo.stop());